src/repository/account_repository.go

```diff
@@ -53,4 +54,28 @@ func (r *accountRepository) RegisterNewAccountToDatabase(ctx context.Context, da
 	return
 }
 
-// TODO : Login and FindById
+func (r *accountRepository) FindByEmail(ctx context.Context, email string) (account *model.Login, err error) {
+	row := sq.Select("id", "email", "password").
+		From("accounts").
+		Where(sq.Eq{"email": email}).
+		RunWith(r.db).
+		QueryRowContext(ctx)
+
+	var data model.Login
+
+	err = row.Scan(
+		&data.ID,
+		&data.Email,
+		&data.Password,
+	)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			logrus.WithField("email", email).Error("email not found")
+			return nil, fmt.Errorf("email %s not found", email)
+		}
```

Comment:
better don't return error if data is not found or empty, just return nil to the front. then let usecase layer consider the flow